xmtp_mls/src/constants.rs

```diff
@@ -0,0 +1 @@
+pub(crate) const HMAC_SALT: &[u8] = b"libXMTP HKDF salt!";
```

Comment:
we have a place for constants in `configuration.rs` github.com/xmtp/libxmtp/blob/main/xmtp_mls/src/configuration.rs

Reply:
Ahh, cool thanks. I'll move it there!